application/console/controllers/CronController.php

```diff
@@ -31,6 +31,12 @@ public function actionRemoveOldUnverifiedRecords()
             'status' => 'starting',
         ]);
         Mfa::removeOldUnverifiedRecords();
+
+        \Yii::warning([
```

Comment:
Putting the "starting" and "complete" log message calls in separate functions makes it possible to call the code in such a way that only one of those messages will actually be logged. (Example: If any code outside of this CronController calls the `Mfa::removeOldManagerMfaRecords()` method, the "starting" log message wouldn't be sent.)

This isn't critical, but you may want to consider putting the "starting" and "complete" log messages in the same function, around the implementation (or sub call) that actually does what they're describing. Just a suggestion.